lock-api/src/main/java/com/palantir/lock/client/metrics/TimeLockFeedbackBackgroundTask.java

```diff
@@ -69,25 +80,52 @@ public static TimeLockFeedbackBackgroundTask create(TaggedMetricRegistry taggedM
     public void scheduleWithFixedDelay() {
         executor.scheduleWithFixedDelay(() -> {
             try {
-                log.info("The TimeLock client metrics for startTransaction endpoint aggregated "
-                                + "over the last 1 minute - {}",
-                        SafeArg.of("startTxnStats", ConjureTimeLockClientFeedback
-                                .builder()
-                                .stats(ImmutableMap.of("conjureTimelockServiceBlocking.startTransactions",
-                                        getEndpointStatsForStartTxn()))
-                                .atlasVersion(versionSupplier.get())
-                                .nodeId(nodeId)
-                                .serviceName(serviceName)
-                                .build()));
+                ConjureTimeLockClientFeedback feedbackReport = ConjureTimeLockClientFeedback
+                        .builder()
+                        .startTransaction(getEndpointStatsForStartTxn())
+                        .leaderTime(getEndpointStatsForLeaderTime())
+                        .atlasVersion(versionSupplier.get())
+                        .nodeId(nodeId)
+                        .serviceName(serviceName)
+                        .build();
+                timeLockClientFeedbackServices
+                        .current()
+                        .forEach(service -> reportClientFeedbackToService(feedbackReport, service));
             } catch (Exception e) {
                 log.warn("A problem occurred while reporting client feedback for timeLock adjudication.", e);
             }
         },
-                timeLockClientFeedbackReportInterval.getSeconds(),
-                timeLockClientFeedbackReportInterval.getSeconds(),
+                TIMELOCK_CLIENT_FEEDBACK_REPORT_INTERVAL.getSeconds(),
+                TIMELOCK_CLIENT_FEEDBACK_REPORT_INTERVAL.getSeconds(),
                 TimeUnit.SECONDS);
     }
 
+    private void reportClientFeedbackToService(ConjureTimeLockClientFeedback feedbackReport,
+            TimeLockClientFeedbackService service) {
+        try {
+            service.reportFeedback(AUTH_HEADER, feedbackReport);
+        } catch (Exception e) {
+            // we do not want this exception to bubble up so that feedback can be reported to other hosts
+            log.warn("Failed to report feedback to TimeLock host.", e);
+        }
+    }
+
+    private EndpointStatistics getEndpointStatsForLeaderTime() {
+        return EndpointStatistics.of(getP99ForLeaderTime(),
+                getOneMinuteRateForLeaderTime());
+    }
+
+    private double getOneMinuteRateForLeaderTime() {
+        return conjureTimelockServiceBlockingMetrics.leaderTime().getOneMinuteRate();
+    }
+
+    private double getP99ForLeaderTime() {
+        return conjureTimelockServiceBlockingMetrics
+                .leaderTime()
+                .getSnapshot()
+                .get99thPercentile();
```

Comment:
nit: You could probably have a function that takes a Timer and gets the 99th percentile (then it can be shared with the StartTransaction ones).